### DISCUSSION TITLE
[Notes] Employee OS Engagement WG - 23/01/24

### DISCUSSION THREAD

**Discussion by jlprat** [todogroup/ospology#237] [OSPO Discussions]
*Authors: Employee OS Engagement WG*
*Date: 23/01/24*
*Subject: WG Meeting Notes*

***

The Employee OS Engagement WG gather its second meeting in 2023 on January Tuesday 24th, to start working on OSPO training courses path for open source teams via: https://github.com/todogroup/todogroup.org/milestones

# Summary

New faces in the meeting so we discussed the history and current state of the art. We collected feedback and expectations from the guide.

We discussed the possibility for this guide to include references to static files and processes a contributor needs to look at before starting (Principle 1), such as CLA, DCO, Code of Conduct, CONTRIBUTING...

**Comment by Blackheart2** [comment]
Embajador ospo mexico
